### DISCUSSION TITLE
Feature Request: Icon View Mode

### DISCUSSION THREAD

**Discussion by Nekkowe** [cboxdoerfer/fsearch#509] [Ideas]
In addition to the existing list view, it would be very nice to have an icon view (and possibly even a compact view a lá the Nemo file manager, to round out all the major types offered by most file managers).

This kind of view mode is very important for looking through large amounts of searched-for images or other media (e.g. for 3D texturing workflows, visual reference for digital art, tabletop RPG map creation, picking out icons...)

A standard way of adjusting zoom in many programs is to hold down the CTRL key and scroll the mouse wheel. 
In Everything (and in Windows Explorer), this input method:
- seamlessly switches between these different view modes, with the detailed list view active at the most "zoomed out" zoom level
- while in icon view, scales the size of the thumbnails according to the zoom level

This might be asking a lot, but it would be very nice to see this behaviour replicated here.

---

Thank you for all the hard work you've put into this amazing program. It's an absolute lifesaver.

**Comment by cboxdoerfer** [comment]
Different views and zooming are definitely something I'm going to add. Currently I'm still working on the new database backend for the 0.3 release and I don't want to distract myself from that too much, but once 0.3 is out I'm going to focus more on the UI (including those features) and new versions will be released more frequently.
